.github/workflows/ci.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install poetry
-        poetry install
+        poetry update
```

Comment:
why this change?

Reply:
oh, well, I see fixes the error that we were receiving, not sure why and how but ok cool